tests/tinytest.R

```diff
@@ -1,5 +1,9 @@
 
-if ( requireNamespace("tinytest", quietly=TRUE) ){
+if (requireNamespace("tinytest", quietly=TRUE) &&
+  utils::packageVersion("tinytest") >= "1.1.0") {
+
+  ## Set a seed to make the tests deterministic
+  set.seed(42)
```

Comment:
Please add

```r
    ## R makes us to this
    Sys.setenv("R_TESTS"="")
```

which other runners set or recommend. I forget the details but this had to do with a nasty to debug issue in running `R CMD check`.

Reply:
It seems my comment got lost: R_TESTS is set by tinytest starting from version 1.1.0